mmpose/datasets/datasets/base/kpt_3d_mview_rgb_img_direct_dataset.py

```diff
@@ -111,11 +114,6 @@ def _get_mapping_id_name(imgs):
 
         return id2name, name2id
 
-    @abstractmethod
```

Comment:
Why remove this method?

Reply:
In campus and shelf datasets, training and testing samples are quite different. Also, there is some randomness while preparing training samples that can not be determined and stored in a fixed database. 
It seems difficult to use a single method like '_get_db' to prepare both train and test samples.